proto/abis/massa/abi/v1/abi.proto

```diff
@@ -130,53 +130,53 @@ message RespResult {
     CheckedSubNativeTimeResult checked_sub_native_time_result = 15;
     CompareAddressResult compare_address_result = 16;
     CompareNativeAmountResult compare_native_amount_result = 17;
     CompareNativeTimeResult compare_native_time_result = 18;
     ComparePubKeyResult compare_pub_key_result = 19;
     CreateScResult create_sc_result = 20;
-    DeleteDataResult delete_data_result = 21;
-    DivRemNativeAmountsResult div_rem_native_amounts_result = 22;
-    FunctionExistsResult function_exists_result = 23;
-    GenerateEventResult generate_event_result = 24;
-    GetAddressCategoryResult get_address_category_result = 25;
-    GetAddressVersionResult get_address_version_result = 26;
-    GetBalanceResult get_balance_result = 27;
-    GetBytecodeResult get_bytecode_result = 28;
-    GetCallCoinsResult get_call_coins_result = 29;
-    GetCallStackResult get_call_stack_result = 30;
-    GetCurrentSlotResult get_current_slot_result = 31;
-    GetDataResult get_data_result = 32;
-    GetKeysResult get_keys_result = 33;
-    GetNativeTimeResult get_native_time_result = 34;
-    GetOpDataResult get_op_data_result = 35;
-    GetOpKeysResult get_op_keys_result = 36;
-    GetOriginOperationIdResult get_origin_operation_id_result = 37;
-    GetOwnedAddressesResult get_owned_addresses_result = 38;
-    GetPubKeyVersionResult get_pub_key_version_result = 39;
-    GetRemainingGasResult get_remaining_gas_result = 40;
-    GetSignatureVersionResult get_signature_version_result = 41;
-    HashBlake3Result hash_blake3_result = 42;
-    HashSha256Result hash_sha256_result = 43;
-    HasDataResult has_data_result = 44;
+    DeleteDsEntryResult delete_ds_entry_result = 21;
+    DivRemNativeAmountResult div_rem_native_amount_result = 22;
+    DsEntryExistsResult ds_entry_exists_result = 23;
+    FunctionExistsResult function_exists_result = 24;
+    GenerateEventResult generate_event_result = 25;
+    GetAddressCategoryResult get_address_category_result = 26;
+    GetAddressVersionResult get_address_version_result = 27;
+    GetBalanceResult get_balance_result = 28;
+    GetBytecodeResult get_bytecode_result = 29;
+    GetCallCoinsResult get_call_coins_result = 30;
+    GetCallStackResult get_call_stack_result = 31;
+    GetCurrentSlotResult get_current_slot_result = 32;
+    GetDsKeysResult get_ds_keys_result = 33;
+    GetDsValueResult get_ds_value_result = 34;
+    GetNativeTimeResult get_native_time_result = 35;
+    GetOpDataResult get_op_data_result = 36;
+    GetOpKeysResult get_op_keys_result = 37;
+    GetOriginOperationIdResult get_origin_operation_id_result = 38;
+    GetOwnedAddressesResult get_owned_addresses_result = 39;
+    GetPubKeyVersionResult get_pub_key_version_result = 40;
+    GetRemainingGasResult get_remaining_gas_result = 41;
+    GetSignatureVersionResult get_signature_version_result = 42;
+    HashBlake3Result hash_blake3_result = 43;
+    HashSha256Result hash_sha256_result = 44;
     HasOpKeyResult has_op_key_result = 45;
```

Comment:
`HasDataResult` became `DsEntryExistsResult`, but we're not changing `HasOpKeyResult`

Reply:
I dunno…
I asked myself the same question but did not come with a good name, so I did not change it (also it implies change here and in 5 other repos, so…)
Propose a name, i'll do the change

Reply:
```GetDsKeysResult``` VS ```GetOpKeysResult ```    
So 
```DsEntryExistsResult``` VS ```OpEntryExistsResult ```    
?

Reply:
done